com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/DeleteServicesStep.java

```diff
@@ -83,23 +89,24 @@ protected String getStepErrorMessage(DelegateExecution context) {
         return Messages.ERROR_DELETING_SERVICES;
     }
 
-    private Map<String, CloudServiceExtended> getServicesData(List<String> serviceNames, ExecutionWrapper execution) {
+    private List<CloudServiceExtended> getServicesData(List<String> serviceNames, ExecutionWrapper execution) {
         CloudControllerClient client = execution.getControllerClient();
 
         return serviceNames.parallelStream()

```

Comment:
Why is the stream parallel?

Reply:
http call to the controller for every service